src/main/java/seedu/address/logic/commands/tag/DeleteCommand.java

```diff
@@ -22,45 +23,60 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the tag identified by the index number used in the displayed reminder list. "
             + "Note that all associations with this tag will be cleared.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Parameters: INDEX (must be a positive integer) MODEL (must be either contact or sales)\n"
```

Comment:
I think this part should be PREFIX_CONTACT_TAG + INDEX or PREFIX_SALE_TAG + INDEX?